### PR TITLE
Add host address

### PR DESCRIPTION
**Pull request by absk1317**
Add host address in the template. Earlier host alias only was being used.

### CODE REVIEW

File: mattermost.py [hunk 1]
```diff
@@ -27,7 +27,7 @@
 VERSION = "0.2.0"
```
Comment (andrefreitas) [inline mattermost.py]: Need to increment version to 0.2.0

**Review by andrefreitas — COMMENTED** [review]
Hi,
Thanks for your PR. Everything is fine for me. Just please increment the version

File: mattermost.py (hunk 1 above)
Comment (absk1317) [reply mattermost.py]: I am sorry, did you mean to increment the version to 0.3.0 as its already at 0.2.0 ?

Comment (andrefreitas) [reply mattermost.py]: Sorry, yes, to 0.3.0 :)

Comment (absk1317) [reply mattermost.py]: Cool. Fixed it. :)

**Pull request merged**